app/src/main/res/layout/feeditemlist_item.xml

```diff
@@ -189,14 +189,14 @@
                     style="@style/AntennaPod.TextView.ListItemSecondaryTitle"
                     tools:text="00:42:23" />
 
-                <ProgressBar
+                <com.google.android.material.progressindicator.LinearProgressIndicator
                     android:id="@+id/progressBar"
                     android:layout_width="0dp"
-                    android:layout_height="4dp"
+                    android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:max="100"
                     android:layout_margin="4dp"
-                    style="?android:attr/progressBarStyleHorizontal" />
+                    tools:background="@android:color/holo_blue_light" />
```

Comment:
This background is no longer needed. Could you please remove it when touching the line anyway?